### PR TITLE
Preparatory tidy up work for json api spike

### PR DESCRIPTION
**Pull request by andrewrlee**
* Removing unused code
* updating video.js
* Extracting out services/index

### Context

> Does this issue have a Trello card?
https://trello.com/c/cuCgbKtD

### Intent

bit of refactoring/tidy up in preparation for json api work

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,89 +1,4 @@
-const redis = require('redis');
-const { path } = require('ramda');
 const { createApp } = require('./app');
-const { logger, requestLogger } = require('./utils/logger');
-const config = require('./config');
+const services = require('./services');
```
Comment (andrewrlee) [inline server/index.js:2]: extracted out service wire up from app creation

File: server/services/hubMenu.js [hunk 2]
```diff
@@ -1,27 +1,6 @@
...
-
-  return {
-    tagsMenu,
-    primaryMenu,
-    categoryMenu,
-    allTopics,
-  };
-};
+const createHubMenuService = repository => ({
+  allTopics: prisonId => repository.allTopics(prisonId),
```
Comment (andrewrlee) [inline server/services/hubMenu.js:2]: these other methods weren't being used - I'll convert this into a `TopicService` class in a follow up

File: server/repositories/hubMenu.js [hunk 3]
```diff
@@ -93,8 +93,6 @@ function hubMenuRepository(httpClient, jsonClient) {
   }
 
   return {
-    tagsMenu,
-    primaryMenu,
```
Comment (andrewrlee) [inline server/repositories/hubMenu.js:97]: these are only used internally.

File: server/repositories/__tests__/hubMenuRepository.spec.js [hunk 4]
```diff
@@ -1,37 +1,7 @@
...
-      const httpClient = {
-        get: jest.fn().mockReturnValue(topicsResponse),
-      };
-
-      const repository = hubMenuRepository(httpClient);
-      const expected = [
-        { linkText: 'Bat', href: '/tags/1', id: '1', description: undefined },
-        { linkText: 'Baz', href: '/tags/0', id: '0', description: undefined },
-      ];
-      const result = await repository.tagsMenu();
```
Comment (andrewrlee) [inline server/repositories/__tests__/hubMenuRepository.spec.js:18]: Deleted these 2 tests, they were testing internal methods.

* This test isn't testing what it claims to be testing.
* The other test is very basic happy path which will be rewritten in a follow up (with additional cases)

**Review by jonhindle — APPROVED** [review]

**Pull request merged**